appengine/flexible/StaticContent/StaticContent.Sample/Program.cs

```diff
@@ -14,22 +14,16 @@
  * the License.
  */
 
-using Microsoft.AspNetCore.Hosting;
-using System.IO;
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
 
-namespace StaticContent
-{
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseStartup<Startup>()
-                .Build();
+// [START gae_flex_dotnet_static_files]

```

Comment:
I was wondering if it's safe to move this tag. From my reading of https://source.corp.google.com/search?q=gae_flex_dotnet_static_files, this should be fine.

Reply:
If you move a tag that's being used Snippet Bot will complain and the Snippet Bot Check will be red, meaning the PR cannot be merged. On this PR Snippet Bot [is not complaining](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2182#issuecomment-1540793623), it's just mentioning that we are removing two tags and adding two tags. So yes, this is safe.